### PR TITLE
feat(vite): add calcite dist output target sample

### PR DESCRIPTION
**Pull request by eriklharper**
Related Issue: https://github.com/Esri/calcite-components/pull/6452

This adds the `dist` output target sample code to the vite example for posterity.

### CODE REVIEW

File: vite/index.html [hunk 1]
```diff
@@ -8,7 +8,16 @@
...
     <calcite-button>My Button</calcite-button>
+    <calcite-date-picker range></calcite-date-picker>
+    <calcite-input></calcite-input>
     <calcite-icon icon="banana"></calcite-icon>
+
     <script type="module" src="/main.js"></script>
+
+    <!--
+      Replace main.js with the script below to use Vite with auto-lazy-loaded Calcite Components
+      @link https://stenciljs.com/docs/distribution
```
Comment (geospatialem) [inline vite/index.html:19]: Could we add a link to our docs site in place of the stencil linkage?
> @link https://developers.arcgis.com/calcite-design-system/get-started/#distribution

File: vite/index.html [hunk 2]
```diff
@@ -8,7 +8,16 @@
   </head>
   <body>
     <calcite-button>My Button</calcite-button>
+    <calcite-date-picker range></calcite-date-picker>
+    <calcite-input></calcite-input>
     <calcite-icon icon="banana"></calcite-icon>
+
     <script type="module" src="/main.js"></script>
```
Comment (geospatialem) [inline vite/index.html:15]: WDYT about:

```
<!-- If using the distribution build, comment out the "main.js" file-->
```

File: vite/index.html [hunk 3]
```diff
@@ -8,7 +8,16 @@
...
+
+    <!--
+      Replace main.js with the script below to use Vite with auto-lazy-loaded Calcite Components
```
Comment (geospatialem) [inline vite/index.html:18]: WDYT about:
> If using the distribution build, load the "calcite-dist-build.js" file to use Vite with auto-lazy loaded Calcite Components.

**Review by geospatialem — APPROVED** [review]
Some small recommendations, otherwise LGTM! Thanks for tackling @eriklharper! 🚀

**Comment by eriklharper** [comment]
@geospatialem all those edits look great!  Just updated them all.

**Pull request merged**